app/screens/settings-screen/settings-screen.tsx

```diff
@@ -69,13 +69,21 @@ export const SettingsScreen: React.FC<Props> = ({ navigation }) => {
 
   const username = data?.me?.username ?? undefined
   const phone = data?.me?.phone ?? undefined
-  const languageQuery = data?.me?.language ?? "DEFAULT"
+  const languageQuery = data?.me?.language
```

Comment:
```suggestion
  const languageQuery = data?.me?.language || "DEFAULT"
```

should do the trick.

But it's the type of use case where `fixing` the backend would be better imo?

Reply:
yup that works @daviroo 

```
$ node
Welcome to Node.js v18.10.0.
Type ".help" for more information.
> let language = undefined
undefined
> language || "Default"
'Default'
> language ?? "Default"
'Default'
> language = ""
''
> language || "Default"
'Default'
> language ?? "Default"
''
```